### PR TITLE
Send Congrats Email on First Membership of the Semester

### LINKED ISSUE #62: Scoreboard: Send Congrats Email on First Membership
We talked about sending an email to people when they get their first membership of the semester congratulating them and explaining what membership gives them.

[Mailgun has 10K free emails a month](https://www.mailgun.com/pricing) which seems like a viable option. 

Acceptance Criteria:
- Integrate w/ Mailgun
- Create email template w/ body copy
- Send the email to users if this is the first membership they are given this semester

### PR DESCRIPTION
**Pull request by billdybas**
Makes the API send a congrats email when a user has their first membership approved for the semester.

Still need to:
- [x] Test that this works
- [x] Make an HTML email w/ our logo and color scheme
- [x] Add Info the README on getting started w/ Mailgun
- [x] Wait for ITS to update our DNS records for Mailgun
- [x] Add the new secrets to our Docker setup
- [ ] Deploy this feature

Closes #62

### CODE REVIEW

**Comment by billdybas** [comment]
@wadeanthony0100 @robmcl4 Here's what the HTML email looks like in Gmail.

**Comment by wadeanthony0100** [comment]
😍

File: emails/scoreboard-welcome.heml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    </style>
+  </head>
+  <body>
+    <container>
+      <block>
+        <img src="" />
+      </block>
+      <block>
+        <p>Hello ,</p>
+        <p>Congratulations! 🎉 You have earned your first SSE membership for this semester.</p>
```
Comment (MatthewMazaika) [inline emails/scoreboard-welcome.heml]: it doesn't appear that your :tada: emoji got persisted

Comment (billdybas) [reply emails/scoreboard-welcome.heml]: [Huh?](https://github.com/rit-sse/node-api/pull/74/files#diff-701d4bea5aff9022464772fc5ee49952R109)

File: app.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import mime from 'mime';
 import nconf from './config';
 import router from './routes';
 import './models';
+import './events';
```
Comment (vuzDylan) [inline app.js]: Why is this here?

Comment (vuzDylan) [reply app.js]: NVM I thought this was the model MB

**Comment by vuzDylan** [comment]
Looks good I'm not sure why bluebird is still in this but sure
